lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module

```diff
@@ -72,6 +72,11 @@ function dosomething_zendesk_form($form, &$form_state, $entity = NULL)  {
     $ip_address = ip_address();
   }
 
+  $form['honeypot'] = [
```

Comment:
Maybe don't call it honeypot? Might the bot know to ignore a field labeled as such?